pkg/irc/irc.go

```diff
@@ -8,11 +8,12 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
+	"strings"
 
 	"github.com/Elemental-IRCd/irc"
 )
 
-const VERSION = "v1.2.0"
+const VERSION = "v1.3.0"
```

Comment:
I'd prefer if the version wasn't bumped in a PR - can I ask that you please remove this line?